main.tf

```diff
@@ -15,6 +15,8 @@ resource "aws_vpc" "default" {
 
   #bridgecrew:skip=BC_AWS_LOGGING_9:VPC Flow Logs are meant to be enabled by terraform-aws-vpc-flow-logs-s3-bucket and/or terraform-aws-cloudwatch-flow-logs
   cidr_block                       = var.cidr_block
+  ipv4_ipam_pool_id                = var.ipv4_ipam_pool_id
+  ipv4_netmask_length              = var.ipv4_netmask_length
```

Comment:
I believe I've updated to execute `examples/complete` without regression... I still need to put together an example to validate the use of IPAM instead of using `cidr_block` as `examples/complete` does. I'll remove `WIP:` from the PR title when I believe it's fully ready with testing completed.